packer/linux/scripts/install-docker.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eu -o pipefail
 
-DOCKER_VERSION=20.10.14
+DOCKER_VERSION=20.10.15
```

Comment:
While we're here, we might as well update it to [latest](https://docs.docker.com/engine/release-notes/#201017), i reckon!
```suggestion
DOCKER_VERSION=20.10.17
```